### ISSUE TITLE
Adversarial example generation by FGSM: different normalization of training vs test images?

### ISSUE THREAD

**Issue by hookxs** [pytorch/tutorials#1032]
In the Adversarial example generation tutorial the classifier from https://github.com/pytorch/examples/tree/master/mnist is used. However, this classifier is trained with input normalization  `transforms.Normalize((0.1307,), (0.3081,))` while in the FGSM tutorial no normalization is used and the perturbed images are clamped to [0,1] - is this not a contradiction?

**Comment by Melcfrn** [comment]
Hi, 
Furthermore, dimensions in convolution layers are differents. 
The file should have been changed. But the link to the model saved is good.

Clément

**Comment by QasimKhan5x** [comment]
If I am understanding correctly, the following changes are needed in this tutorial:

1. The `Net` in the [Adversarial example generation](https://pytorch.org/tutorials/beginner/fgsm_tutorial.html) tutorial does not match the one in [MNIST]() tutorial. The model definition needs to be changed.
2. The Adversarial example generation does not apply `transforms.Normalize((0.1307,), (0.3081,))` in its `test_loader`, whereas the dataloaders in the MNIST example use it. Therefore, I should add this transform.

@svekars if I'm understanding this correctly, may I make these changes and proceed with a pull request?
